### DISCUSSION TITLE
Does anyone have an updated ddt2000 database for December 2023?

### DISCUSSION THREAD

**Discussion by Deswa16** [cedricp/ddt4all#1252] [Q&A]
Does anyone have the current 4.48 version of the ddt2000 program and the current database for December 2023?

**Comment by RenaultGeek** [comment]
did you manage to find it?

**Comment by CrashOverride2** [comment]
It is very easy to find it, just google it, on the first page you get a forum which you can download it.

But there are only a few files which are newer than 11.2021 so not really worth it.
![ddt](https://github.com/user-attachments/assets/e6ca9bf3-7849-4055-b294-688bae201a17)
